pupil_src/shared_modules/video_capture/file_backend.py

```diff
@@ -30,11 +31,13 @@
 
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
-
-assert av.__version__ >= "0.4.2", "pyav is out-of-date, please update"
 av.logging.set_level(av.logging.ERROR)
 logging.getLogger("libav").setLevel(logging.ERROR)
 
+assert av.__version__ >= "0.4.2", "pyav is out-of-date, please update"
+assert sys.version_info >= (3, 7), \
+    ("We need python 3.7 or later to run Pupil player")
```

Comment:
TODO: Make PR compatible with Python 3.5, i.e. remove dataclasses and type annotations.